draft-ietf-rtcweb-jsep.xml

```diff
@@ -1980,6 +1980,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <xref target="I-D.ietf-rtcweb-video"></xref>(see Section
               5).</t>
 
+              <t>If codec preferences have been set for the associated
+              transceiver, media formats MUST be generated in the
+              corresponding order.</t>
+
```

Comment:
This is a good argument. You've convinced me we should apply this for initial offers and answers (which sounds like the decision the W3C came to).